schemas/top-api.yml

```diff
@@ -254,6 +255,46 @@ paths:
           $ref: '#/components/responses/UnexpectedError'
       tags:
         - conceptcluster
+  /conceptcluster/corpora:
+    get:
+      operationId: getCorpusIds
+      summary: Searches for specific concept clusters
+      responses:
+        '200':
+          description: Returns a distinct list of all the 'corpusIds' found in Concept Nodes.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/StringArray'
+        '401':
+          $ref: '#/components/responses/Unauthorized'
+        default:
+          $ref: '#/components/responses/UnexpectedError'
+      tags:
+        - conceptcluster
+  /conceptcluster/processes:
```

Comment:
Can we integrate this into the already existing paths for pipeline management. I would like to abstract the inner logic of the pipeline.

Reply:
I see, yes, I'm on it.